src/System.ComponentModel.Annotations/tests/ValidationExceptionTests.cs

```diff
@@ -78,5 +82,27 @@ public static void Ctor_String_ValidationAttribute_Object(string errorMessage)
             Assert.Same(validatingAttribute, ex.ValidationAttribute);
             Assert.Same(value, ex.Value);
         }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void Deserialize_ValidBytes_ThrowsPlatformNotSupportedException()
+        {
+            using (var stream = new MemoryStream())
+            {
+                var formatter = new BinaryFormatter();
+                formatter.Serialize(stream, new SubException());
+
+                stream.Position = 0;
+                Exception ex = Assert.Throws<TargetInvocationException>(() => formatter.Deserialize(stream));
+                Assert.IsType<PlatformNotSupportedException>(ex.InnerException);
+            }
+        }
+
+        [Serializable]
+        public class SubException : ValidationException
```

Comment:
ValidationException is not `[Serializable]` so can't be serialized. The test is deserialization, so obviously if we can't serialize, then we can't test deserialization